src/fmu/pem/pem_utilities/pem_config_validation.py

```diff
@@ -284,12 +280,34 @@ class Gas(BaseModel):
         le=0.87,
         description="Gas gravity is a ratio of gas molecular weight to that air",
     )
-    model: str = Field(
+    model: GasModels = Field(
         default="HC2016",
         description="Gas model is one of 'Global', 'Light', or 'HC2016' (default)",
     )
 
 
+class MixModelWood(BaseModel):
+    method: SkipJsonSchema[FluidMixModel] = "wood"
+
+
+class MixModelBrie(BaseModel):
+    method: SkipJsonSchema[FluidMixModel] = "brie"
+    brie_exponent: float = Field(
+        default=3.0,
+        description="Brie exponent selects the mixing curve shape, from linear mix to "
+        "harmonic mean",
+    )
```

Comment:
Are there theoretical bounds (min and/or max) on the brie exponent we can add to validation?